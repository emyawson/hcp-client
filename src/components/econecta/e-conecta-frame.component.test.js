import React from 'react';
import { shallow } from 'enzyme';

import { EConectaFrame } from './e-conecta-frame.component';

describe('EC6 Iframe Component Test', () => {
  const mockProps = {
    action: 'test',
    token: '123456',
  };

  it('should render correctly', () => {
    const wrapper = shallow(<EConectaFrame {...mockProps} />);
    expect(wrapper).toMatchSnapshot();
  });
});
