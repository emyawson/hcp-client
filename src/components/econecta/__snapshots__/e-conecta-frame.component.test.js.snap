// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EC6 Iframe Component Test should render correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <EConectaFrame
    action="test"
    height="calc(100vh - 110px)"
    onLoad={[Function]}
    token="123456"
    width="100%"
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": null,
    "nodeType": "class",
    "props": Object {
      "frameBorder": "0",
      "height": "calc(100vh - 110px)",
      "onLoad": [Function],
      "resizeOnLoad": true,
      "scrolling": "no",
      "src": "/post-redirect.html",
      "styleProps": Object {},
      "width": "100%",
    },
    "ref": null,
    "rendered": null,
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": null,
      "nodeType": "class",
      "props": Object {
        "frameBorder": "0",
        "height": "calc(100vh - 110px)",
        "onLoad": [Function],
        "resizeOnLoad": true,
        "scrolling": "no",
        "src": "/post-redirect.html",
        "styleProps": Object {},
        "width": "100%",
      },
      "ref": null,
      "rendered": null,
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
